### DISCUSSION TITLE
[Feature Request] Add Personalised difficulity level of the problems

### DISCUSSION THREAD

**Discussion by vikramsaha7** [Algolisted-Org/AlgoListed#96] [Ideas]
Just before the interview we don't get enough time to revise all the problems in that case we want to revise only those problems which are hard for us to solve or come up with immediate solution. It would be great if you could add a difficulty level just beside the'mark as done' option. Now when a user go through the problem he/she can mark difficulty level of the problem according to him/her, so later when the user want to have a quick revision he/she can go through only hard problems.

![dif level](https://github.com/Nayaker/AlgoListed/assets/26772559/922ce521-f08b-4b3c-ae43-fd9bbff87a33)

**Comment by NayakPenguin** [comment]
Yeah, that is a good idea to put the self-rating of the problems. 
But don't you think the redo button on the right means the same? I have kept that button for the reason that people can mark the problems that they want to do again. 
_What your take on it as a user ?_
